tests/generate.py

```diff
@@ -210,7 +210,7 @@ def disable(self, excluded):
                     module.modified = True
 
     def suite_count(self):
-        return len(self.modules)
+        return sum(len(module.initializers) for module in self.modules.values())
```

Comment:
> Are we skipping over modules which have no initializers at all now?

Sounds like it does, yes. `min(☝️, 1)` then ?

Reply:
I have no such test suites in `libssh2`, which explains why I missed it 😞.

Reply:
Yeah, I tried the same locally, but in fact this didn't fix the issue for me. Didn't dig deeper yet, though

Reply:
Oops. Because we both had the same issue of using `min` instead of `max`. See #5175 for a fix